client/package.json

```diff
@@ -43,8 +42,8 @@
     "reqwest": "2.0.5",
     "reselect": "3.0.1",
     "reselect-map": "1.0.3",
-    "styled-components": "^2.2.1",
-    "weaveworks-ui-components": "git+https://github.com/weaveworks/ui-components.git#v0.1.51",
+    "styled-components": "2.2.1",
+    "weaveworks-ui-components": "git+https://github.com/weaveworks/ui-components.git#v0.2.0",
```

Comment:
We need to merge https://github.com/weaveworks/scope/pull/2919 first because of Time Travel API changes in `v0.1.52`. Will test all the behaviour more thoroughly once that's done.

Reply:
Alternatively, just copying the changes from the TT wrapper https://github.com/weaveworks/scope/pull/2919/files#diff-d79edc7959396ddc637201c514c8638b would probably do the trick.